controller/work_item_children_blackbox_test.go

```diff
@@ -814,6 +815,93 @@ func lookupWorkitem(t *testing.T, wiList app.WorkItemList, wiID uuid.UUID) *app.
 			return wiData
 		}
 	}
-	t.Error(fmt.Sprintf("Failed to look-up work item with id='%s'", wiID))
+	t.Errorf("Failed to look-up work item with id='%s'", wiID)
 	return nil
 }
+
+func lookupWorkitemFromSearchList(t *testing.T, wiList app.SearchWorkItemList, wiID uuid.UUID) *app.WorkItem {
+	for _, wiData := range wiList.Data {
+		if *wiData.ID == wiID {
+			return wiData
+		}
+	}
+	t.Errorf("Failed to look-up work item with id='%s'", wiID)
```

Comment:
@hectorj2f If the lookup fails, there is no point in a proceeding; better to fail at once. So, I will change it to `Fatalf`